web_src/js/index.js

```diff
@@ -4033,3 +4033,13 @@ $(document).on('click', '.oauth-login-image', () => {
     oauthNav.show();
   }, 5000);
 });
+
+// loading-button this logic used to prevent push one form more than one time
+$(document).on('click', '.button.loading-button', function () {
+  const $btn = $(this);
+  $btn.addClass('loading');
+});
+
+$(document).on('click', '.button.loading-button.loading', (e) => {
+  e.preventDefault();
```

Comment:
It would be better to move this check to previous button click event and just check if button has class loading than prevent default and return